api/src/test/java/org/openmrs/api/ObsServiceTest.java

```diff
@@ -540,6 +535,22 @@ public void getHandler_shouldHaveDefaultImageAndTextHandlersRegisteredBySpring()
 		Assert.assertNotNull(textHandler);
 	}
 	
+	@Test
+	@Verifies(value = "should purge complex obs when complex data file is missing from disk", method = "purgeObs(Obs)")
+	public void purgeComplexObs_shouldSucceedWhenCompleDataFileMissingFromDisk() throws Exception {
```

Comment:
Hi @teleivo , thanks for the tip.
So I did all that and the boiletplate test had already been generated and implemented. However it wasn't handling my case about complex obs. So I added my case in it.
Repushing now...